src/main/java/com/dougnoel/sentinel/elements/PageElement.java

```diff
@@ -6,6 +6,7 @@
 import java.time.Duration;
```

Comment:
You ran this file through a coding standards tool, and while I appreciate the thought there are some things in here I agree with and some I don't. Please revert the file and only check in your changes. Running code through a tool like this requires discussion. This file is a core file and I need to go through it with a fine-tooth comb. That is difficult to do when there are so many spacing changes distracting from your code. If you want to do a cleanup like this, we should first discuss the rules, and second these changes should be made in a PR without any functional changes.